pcdsdevices/lodcm.py

```diff
@@ -1295,6 +1315,11 @@ def __init__(self, prefix, *, name, main_line='MAIN', mono_line='MONO',
         self.h2nC = self.tower2.h2nC
         self.h2nSi = self.tower2.h2nSi
 
+        # energy aliases
+        self.E = self.energy
```

Comment:
I'm sure you're in the process of debugging this, but this assignment fails during testing because our test suite doesn't set up the energies in time for the device init.  

My interactive testing is currently failing due to the lodcm not being fully configured at the moment, but something like this could at least appease the test suite:
```python
@property
def E(self)
    return self.energy
```